examples/README.md

```diff
@@ -0,0 +1,76 @@
+# Zenoh-Flow examples
+
+## How to run
+
+### Build
+
+We can create all the zenoh-flow node libraries used in the examples with the following command:
+   ```bash
+  cargo build --examples
+   ```
+
+Alternatively, we can create a single library of a zenoh-flow node with the following command:
+   ```bash
+  cargo build --example name_node 
+
+  i.e.
+  cargo build --example greetings-maker
+   ```
+
+### Update the paths
+
+For each YAML file in the list below, check that the paths and filenames are
+correct:
+- flows/period-miss-detector.yaml 
+- flows/getting-started.yaml
+- examples/file-writer/file-writer.yaml
+- examples/greetings-maker/greetings-maker.yaml
+- examples/period-miss-detector/period-miss-detector.yaml
+
+### Launch
+
+#### 1st terminal: Zenoh
+
+```shell
+cd ~/dev/zenoh && ./target/debug/zenohd -c ~/.config/zenoh-flow/zenoh.json
+```
+
+#### 2nd terminal: Zenoh-Flow daemon
+
+```shell
+cd ~/dev/zenoh-flow/ && ./target/debug/zenoh-flow-daemon -c ~/.config/zenoh-flow/runtime.yaml
+```
```

Comment:
Merge these 2 paragraphs into one, and point to the documentation: https://github.com/eclipse-zenoh/zenoh-flow/wiki/Installation-(v0.4.0)#zenoh-plugin